### DISCUSSION TITLE
Feature Request: Add Sorting Options to Taxonomy Filter in Elementor

### DISCUSSION THREAD

**Discussion by JennyPower** [elementor/elementor#24791] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

I would like to propose a feature enhancement for Elementor - the addition of sorting options in the taxonomy filter. Currently, Elementor's taxonomy filter provides robust functionality for filtering content based on taxonomies. However, the ability to sort these filters (e.g., alphabetically, by count, or custom order) is not available. This feature would greatly enhance user experience and flexibility, allowing for more structured and user-friendly navigation, especially in sites with extensive content categories.

### Describe the solution you'd like.

For websites with a large number of categories, tags, or custom taxonomies, navigating through the taxonomy filter can be overwhelming for users. Providing sorting options would make it easier for users to find the specific category or tag they are interested in. For example, in an e-commerce site, sorting product categories alphabetically or by popularity could significantly improve the shopping experience.

### Describe alternatives you've considered.

Alphabetical Sorting: Enable sorting of taxonomy terms in ascending or descending alphabetical order.
Count-Based Sorting: Sort taxonomy terms based on the number of posts associated with each term, either in ascending or descending order.
Custom Order: Allow users to manually set the order of taxonomy terms as per their preference.

### Additional context

This enhancement aligns with Elementor's commitment to providing user-friendly and highly customizable web design tools. The proposed feature would not only benefit end-users but also website developers and designers who strive to create more engaging and efficient user interfaces.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by DanielCyran** [comment]
great idea

**Comment by Emmilemmi** [comment]
I'd appreciate a solution

**Reply by ProtoUI** [reply]
For custom order, this plugin helped me : https://fr.wordpress.org/plugins/taxonomy-terms-order/

**Reply by Emmilemmi** [reply]
> I'd appreciate a solution

A piece of CSS helped me! See this video https://youtu.be/jcOJBBbslV4?si=sO9ttUkmYsksdJSc
